lib/src/resources/movie_api_provider.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart';
+import '../model/movie.dart';
+
+class MovieApiProvider {
+  Client client = Client();
+  final _apiKey = "1124adc80a8b854f1da261e38d081ce3";
+  final _baseUrl = "http://api.themoviedb.org/3/movie";
+
+  Future<Movie> fetchMovieList() async {
+    final response = await client.get(Uri.parse(
+        _baseUrl + "/popular?api_key=" + _apiKey + "&language=en-US&page=1"));
```

Comment:
you can add all this strings into a strings constants file, to avoid hardcodes

Reply:
Fixed